packages/webapp/src/routes/deployment/+page.svelte

```diff
@@ -0,0 +1,3174 @@
+<script lang="ts">
+	import { DropdownRadio } from '@rainlanguage/ui-components';
+	import {
+		DotrainOrderGui,
+		// type AddOrderCalldataResult,
+		// type AllowancesResult,
+		type ApprovalCalldataResult,
+		type AvailableDeployments,
+		type DepositAndAddOrderCalldataResult,
+		// type DepositCalldataResult,
+		type GuiDeposit,
+		type GuiFieldDefinition,
+		type TokenInfos
+	} from '@rainlanguage/orderbook/js_api';
+	import { onMount } from 'svelte';
+	import { Button, Input, Label } from 'flowbite-svelte';
+	import { ethers } from 'ethers';
+
+	let dotrain = `
```

Comment:
instead of committing this into the repo, can we load it via http from the rain.webapp repo?